server/src/test/java/org/elasticsearch/cluster/metadata/MetaDataMappingServiceTests.java

```diff
@@ -113,4 +113,17 @@ public void testMappingVersionUnchanged() throws Exception {
         assertThat(result.resultingState.metaData().index("test").getMappingVersion(), equalTo(previousVersion));
     }
 
+    public void testGetRequestIndex() {
```

Comment:
Given that we have the comma concatenating logic, I think we can omit the test here.

Reply:
Of course